### ISSUE TITLE
Norwegian locale missing?

### ISSUE THREAD

**Issue by rasmusrim** [moment/luxon#1006]
**Describe the bug**
No version of the "Norwegian - bokmål" locale gives the correct long month name. It only outputs M05 etc.

**To Reproduce**
```
import React from "react";
import { render } from "react-dom";
import { DateTime } from "luxon";

const App = () => {
  const isoString = "2018-05-01T13:44:48.708709Z";

  const locales = ["no", "nb", "nob", "no-nb", "nb-no", "NOB"];

  return (
    <div>
      {locales.map((locale) => (
        <li>
          {DateTime.fromISO(isoString).setLocale(locale).toFormat("MMMM")}
        </li>
      ))}
    </div>
  );
};

render(<App />, document.getElementById("root"));
```

https://codesandbox.io/s/luxon-playground-forked-b84vu?file=/src/index.js

**Actual vs Expected behavior**
I expect "mai" instead of M05 in at least one of the locales.

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome 92
- Luxon version: 2.0.2
- Your timezone: Europe/Oslo

**Comment by rasmusrim** [comment]
@dasa Thanks a bunch! I'll just wait for the update then.

**Comment by dasa** [comment]
Fixed in 92.0.4515.159
